Cannot npm install due to gpu.js/angle not compiling

```
$  npm install js-slang

npm ERR! code 1
npm ERR! path /xxxx/xxxx/node_modules/gpu.js/node_modules/gl
npm ERR! command failed
npm ERR! command sh -c prebuild-install || node-gyp rebuild
npm ERR! make: Entering directory '/xxxx/xxxx/node_modules/gpu.js/node_modules/gl/build'
npm ERR!   CXX(target) Release/obj.target/angle_common/angle/src/common/Float16ToFloat32.o
npm ERR! make: Leaving directory '/xxxx/xxxx/node_modules/gpu.js/node_modules/gl/build'
npm ERR! prebuild-install warn install No prebuilt binaries found (target=18.16.0 runtime=node arch=x64 libc= platform=linux)
npm ERR! gyp info it worked if it ends with ok
npm ERR! gyp info using node-gyp@9.3.1
npm ERR! gyp info using node@18.16.0 | linux | x64
npm ERR! gyp info find Python using Python version 3.11.3 found at "/usr/bin/python3"
npm ERR! gyp info spawn /usr/bin/python3
npm ERR! gyp info spawn args [
npm ERR! gyp info spawn args   '/xxxx/xxxx/node_modules/node-gyp/gyp/gyp_main.py',
npm ERR! gyp info spawn args   'binding.gyp',
npm ERR! gyp info spawn args   '-f',
npm ERR! gyp info spawn args   'make',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   '/xxxx/xxxx/node_modules/gpu.js/node_modules/gl/build/config.gypi',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   '/xxxx/xxxx/node_modules/node-gyp/addon.gypi',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   '/xxxx/xxxx/.cache/node-gyp/18.16.0/include/node/common.gypi',
npm ERR! gyp info spawn args   '-Dlibrary=shared_library',
npm ERR! gyp info spawn args   '-Dvisibility=default',
npm ERR! gyp info spawn args   '-Dnode_root_dir=/xxxx/xxxx/.cache/node-gyp/18.16.0',
npm ERR! gyp info spawn args   '-Dnode_gyp_dir=/xxxx/xxxx/node_modules/node-gyp',
npm ERR! gyp info spawn args   '-Dnode_lib_file=/xxxx/xxxx/.cache/node-gyp/18.16.0/<(target_arch)/node.lib',
npm ERR! gyp info spawn args   '-Dmodule_root_dir=/xxxx/xxxx/node_modules/gpu.js/node_modules/gl',
npm ERR! gyp info spawn args   '-Dnode_engine=v8',
npm ERR! gyp info spawn args   '--depth=.',
npm ERR! gyp info spawn args   '--no-parallel',
npm ERR! gyp info spawn args   '--generator-output',
npm ERR! gyp info spawn args   'build',
npm ERR! gyp info spawn args   '-Goutput_dir=.'
npm ERR! gyp info spawn args ]
npm ERR! gyp info spawn make
npm ERR! gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
npm ERR! In file included from ../angle/src/common/debug.h:16,
npm ERR!                  from ../angle/src/common/mathutil.h:12,
npm ERR!                  from ../angle/src/common/Float16ToFloat32.cpp:9:
npm ERR! ../angle/src/common/angleutils.h:36:14: error: ‘uintptr_t’ does not name a type
npm ERR!    36 | extern const uintptr_t DirtyPointer;
npm ERR!       |              ^~~~~~~~~
npm ERR! ../angle/src/common/angleutils.h:21:1: note: ‘uintptr_t’ is defined in header ‘<cstdint>’; did you forget to ‘#include <cstdint>’?
npm ERR!    20 | #include <vector>
npm ERR!   +++ |+#include <cstdint>
npm ERR!    21 | 
npm ERR! make: *** [angle/src/angle_common.target.mk:159: Release/obj.target/angle_common/angle/src/common/Float16ToFloat32.o] Error 1
npm ERR! gyp ERR! build error 
npm ERR! gyp ERR! stack Error: `make` failed with exit code: 2
npm ERR! gyp ERR! stack     at ChildProcess.onExit (/xxxx/xxxx/node_modules/node-gyp/lib/build.js:203:23)
npm ERR! gyp ERR! stack     at ChildProcess.emit (node:events:513:28)
npm ERR! gyp ERR! stack     at ChildProcess._handle.onexit (node:internal/child_process:291:12)
npm ERR! gyp ERR! System Linux 6.3.1-arch1-1
npm ERR! gyp ERR! command "/xxxx/xxxx/.nvm/versions/node/v18.16.0/bin/node" "/xxxx/xxxx/node_modules/.bin/node-gyp" "rebuild"
npm ERR! gyp ERR! cwd /xxxx/xxxx/node_modules/gpu.js/node_modules/gl
npm ERR! gyp ERR! node -v v18.16.0
npm ERR! gyp ERR! node-gyp -v v9.3.1
npm ERR! gyp ERR! not ok

```

--- a/package.json
+++ b/package.json
@@ -40,7 +40,7 @@
     "acorn-loose": "^8.0.0",
     "acorn-walk": "^8.0.0",
     "astring": "^1.4.3",
-    "gpu.js": "^2.10.4",
+    "gpu.js": "^2.16.0",
     "js-base64": "^3.7.5",
     "lodash": "^4.17.20",
     "node-getopt": "^0.3.2",

